schema.sql

```diff
@@ -1,5 +1,8 @@
-/* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
-    name varchar(100)
+    id INT PRIMARY KEY,
```

Comment:
- [ ] [optional] You could also make sure the id is never null as its the primary key and missing it might cause an error in the database esp when used to connect with other tables.

Reply:
Hey @danmainah , as we declare a variable as a primary key, it's already a combination of NOT NULL constraint and UNIQUE constraint.